lib/plugins/aws/info/display.js

```diff
@@ -68,11 +68,15 @@ module.exports = {
 
     message += functionsMessage;
 
-    // when verbose info is requested, add the stack outputs to the output
-    if (this.options.verbose) {
+    const hasCustomOutput = _.some(this.gatheredData.outputs, output => output.CustomOutput);
+
+    // when verbose info is requested, display all the stack outputs
+    if (hasCustomOutput || this.options.verbose) {
```

Comment:
If there are no custom options, verbose still outputs the built-in serverless outputs. I've kept that feature, but not sure if this is the expected behaviour.